cabal-install/src/Distribution/Client/ProjectPlanning.hs

```diff
@@ -1717,14 +1717,18 @@ elaborateInstallPlan
               -- invocation of the same `./configure` script.
               -- See https://github.com/haskell/cabal/issues/4548
               --
-              -- Moreover, at this point in time, only non-Custom setup scripts
-              -- are supported.  Implementing per-component builds with
-              -- Custom would require us to create a new 'ElabSetup'
+              -- We have to disable per-component at this point in time, only
+              -- Simple and Configure build types are supported.  Custom and
+              -- Hooks are not implemented.  Implementing per-component builds
+              -- with Custom would require us to create a new 'ElabSetup'
```

Comment:
The first sentence of the two paragraphs (line 1715 and 1720) is repeated, we can probably simplify it:

```suggestion
              -- Custom and Hooks are not implemented.
              -- Implementing per-component builds
              -- with Custom would require us to create a new 'ElabSetup'
```

Reply:
Amended.